src/storage/src/hummock/store/version.rs

```diff
@@ -503,18 +503,23 @@ impl HummockVersionReader {
             staging_iters.push(HummockIteratorUnion::First(imm.into_forward_iter()));
         }
         let mut staging_sst_iter_count = 0;
+        // encode once
+        let prefix_hint = if let Some(prefix) = read_options.prefix_hint.as_ref() {
```

Comment:
Can we use Some(...) and None for prefix_hint?

Reply:
+1